fe/fe-core/src/main/java/com/starrocks/catalog/DiskInfo.java

```diff
@@ -44,7 +44,11 @@
     private static final long DEFAULT_CAPACITY_B = 1024 * 1024 * 1024 * 1024L; // 1T
 
     private String rootPath;
+    // disk capacity
     private long totalCapacityB;
+    // dataTotalCapacityB: dataUsedCapacityB + diskAvailableCapacityB
+    // no need to persist
```

Comment:
It seems the dataTotalCapacityB is non-sense. Could the appearance can be replaced by  dataUsedCapacityB+diskAvailableCapacityB?

Reply:
ok